conf/app.ini

```diff
@@ -120,6 +120,8 @@ SSH_ROOT_PATH =
 SSH_KEY_TEST_PATH =
 ; Path to ssh-keygen, default is 'ssh-keygen' and let shell find out which one to call.
 SSH_KEYGEN_PATH = ssh-keygen
+; Disable SSH Authorized Key Backup when rewriting all keys, default is false
+SSH_DISABLE_AUTHORIZED_KEYS_BACKUP = false
```

Comment:
I don't really like negations in configs. Would not `SSH_ENABLE_AUTHORIZED_KEYS_BACKUP = true` make more sense?